datasets/sst/README.md

```diff
@@ -0,0 +1,163 @@
+- sentiment-scoring
+---
+
+# Dataset Card for sst
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://nlp.stanford.edu/sentiment/index.html
+- **Repository:** [Needs More Information]
+- **Paper:** [Recursive Deep Models for Semantic Compositionality Over a Sentiment Treebank](https://www.aclweb.org/anthology/D13-1170/)
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** [Needs More Information]
+
+### Dataset Summary
+
+The Stanford Sentiment Treebank is the first corpus with fully labeled parse trees that allows for a complete analysis of the compositional effects of sentiment in language.
+
+### Supported Tasks and Leaderboards
+
+- `sentiment-scoring`: Each complete sentence is annotated with a `float` label that indicates its level of positive sentiment from 0.0 to 1.0. One can decide to use only complete sentences or to include the contributions of the sub-sentences (aka phrases). The labels for each phrase are included in the `dictionary` configuration. To obtain all the phrases in a sentence we need to visit the parse tree included with each example. In contrast, the `ptb` configuration explicitly provides all the labelled parse trees in Penn Treebank format. Here the labels are binned in 5 bins from 0 to 4.
+- `sentiment-classification`: We can transform the above into a binary sentiment classification task by rounding each label to 0 or 1.
+
+### Languages
+
+The text in the dataset is in English
+
+## Dataset Structure
+
+### Data Instances
+
+For the `default` configuration:
+{'label': 0.7222200036048889,
+ 'sentence': 'Yet the act is still charming here .',
+ 'tokens': 'Yet|the|act|is|still|charming|here|.',
+ 'tree': '15|13|13|10|9|9|11|12|10|11|12|14|14|15|0'}
+
+For the `dictionary` configuration:
+{'label': 0.7361099720001221, 
+'phrase': 'still charming'}
+
+For the `ptb` configuration:
+{'ptb_tree': '(3 (2 Yet) (3 (2 (2 the) (2 act)) (3 (4 (3 (2 is) (3 (2 still) (4 charming))) (2 here)) (2 .))))'}
```

Comment:
````suggestion
For the `default` configuration:
```
{'label': 0.7222200036048889,
 'sentence': 'Yet the act is still charming here .',
 'tokens': 'Yet|the|act|is|still|charming|here|.',
 'tree': '15|13|13|10|9|9|11|12|10|11|12|14|14|15|0'}
```

For the `dictionary` configuration:
```
{'label': 0.7361099720001221, 
'phrase': 'still charming'}
```

For the `ptb` configuration:
```
{'ptb_tree': '(3 (2 Yet) (3 (2 (2 the) (2 act)) (3 (4 (3 (2 is) (3 (2 still) (4 charming))) (2 here)) (2 .))))'}
```
````